main.go

```diff
@@ -683,11 +683,30 @@ func main() {
 
 	log.Printf("* xcodebuild_version: %s (%s)", xcodebuildVersion.Version, xcodebuildVersion.BuildVersion)
 
-	xcprettyVersion, err := cmd.GetXcprettyVersion()
-	if err != nil {
-		log.Warnf("Failed to get the xcpretty version! Error: %s", err)
-	} else {
-		log.Printf("* xcpretty_version: %s", xcprettyVersion)
+	// Detect xcpretty version
+	if configs.OutputTool == "xcpretty" {
+		fmt.Println()
+		log.Infof("Checking output tool")
+		installed, err := IsXcprettyInstalled()
+		if err != nil {
+			fail("Failed to check if xcpretty is installed, error: %s", err)
+		}
+
+		if !installed {
+			log.Warnf(`🚨  xcpretty is not installed`)
```

Comment:
please do not use icons in the step logs, current we have (or moving into the way to have) unified logging (blue info, yellow warning, red error,...), please do not introduce ad hoc log schemes.